### PR TITLE
Enable Accelerator

### PR DESCRIPTION
**Pull request by paulbalaji**
#### Description

Provides `ACCELERATOR_ENDPOINT` argument to agents and provides the relevant arguments to use either the local/cloud Accelerators for mac/windows agents.

Depends on https://github.com/spatialos/gdk-for-unity-shared-ci/pull/72

#### Tests

Ran the pipeline. Multiple times.

#### Documentation

n/a

### CODE REVIEW

File: .buildkite/premerge.steps.yaml [hunk 1]
```diff
@@ -76,19 +76,25 @@ steps:
     <<: *windows
     artifact_paths:
       - logs/**/*
+    env:
+      ACCELERATOR_ENDPOINT: "unity-accelerator.production-intinf-eu1.i8e.io"
```
Comment (zeroZshadow) [inline .buildkite/premerge.steps.yaml:80]: Can these be set on queue level? So we set it once per windows/mac ?

Comment (paulbalaji) [reply .buildkite/premerge.steps.yaml:80]: We could probably set this in the `pre-command` hook, make use of `BUILDKITE_AGENT_META_DATA_PLATFORM` that the agent will know

Comment (jamiebrynes7) [reply .buildkite/premerge.steps.yaml:80]: I feel like you can set the env for a given agent queue.

cc @petemounce ?

Comment (paulbalaji) [reply .buildkite/premerge.steps.yaml]: Moved it up, going to test and see if this works

Comment (paulbalaji) [reply .buildkite/premerge.steps.yaml]: You can set it, but it'll be wiped if you define env per step:

see the environment tab on the `build` steps here: https://buildkite.com/improbable/gdk-for-unity-premerge/builds/4019

**Pull request merged**